project4/proto/consistent_hash_msgs.proto

```diff
@@ -1,7 +1,16 @@
 syntax = "proto3";
 
-package pub_sub_messages;
+package consistent_hash_msgs;
 
+/// basic server information
+message ServerInfo {
+  /// the id of the name server's new predecessor
```

Comment:
You probably want these docs to be more general now. They should probably just refer to "the server" as opposed to predecessors and successors.

Reply:
good catch; to be honest I just did a copy/paste and kinda forgot about it...